plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/TokenFetchingMetastoreClientFactory.java

```diff
@@ -70,15 +73,25 @@ public ThriftMetastoreClient createMetastoreClientFor(Optional<ConnectorIdentity
         String username = identity.map(ConnectorIdentity::getUser)
                 .orElseThrow(() -> new IllegalStateException("End-user name should exist when metastore impersonation is enabled"));
 
-        String delegationToken;
         try {
-            delegationToken = delegationTokenCache.getUnchecked(username);
+            return clientProvider.createMetastoreClient(Optional.of(getDelegationToken(username)));
+        }
+        catch (TException e) {
+            log.debug(e, "It may be a delegation token permission failure, so try again with a new delegation token");
```

Comment:
I do not think we need this log here. Trino will not be run with `debug` log level enabled anyway and the log message does not provide any actionable information..

Reply:
Thank for giving feedbacks :)
I also agree with what you said. this log has been removed.